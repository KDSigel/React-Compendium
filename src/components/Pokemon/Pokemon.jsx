import React from 'react'

export default function Pokemon({pokemon, hp}) {
    return (
        <div>
            <p>{pokemon}</p>
            <p>{hp}</p>
        </div>
    )
}
