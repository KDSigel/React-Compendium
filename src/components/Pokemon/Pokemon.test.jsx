import { screen, render } from "@testing-library/react";
import Pokemon from "./Pokemon";

it('testing my pokemon', () => {
  render(
    <Pokemon 
      image="http://assets.pokemon.com/assets/cms2/img/pokedex/full/012.png"
      pokemon='butterfree' hp='60' height='11' attack='45' defense='50' 
    />
  );
    const singlePokemon = screen.getByText('Name: butterfree')
    expect(singlePokemon).toMatchSnapshot()
});