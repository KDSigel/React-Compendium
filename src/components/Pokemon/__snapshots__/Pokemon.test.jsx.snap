// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing my pokemon 1`] = `
<p>
  Name: 
  butterfree
</p>
`;
